src/sinks/aws_s_s/sns/mod.rs

```diff
@@ -1,7 +1,7 @@
 mod config;
 mod integration_tests;
 
-pub use super::{
+pub(super) use super::{
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏So in this and the other file, there isn't really value in having `pub(super) use super::<...>` , because the visibility of the stuff in super, already has visibility to itself. It's redundant essentially. Moreover I didn't notice this earlier but, this actually shouldn't have been `pub` to begin with. The import of the stuff from `super` (the parent module), we only want to access from within this module.

```suggestion
use super::{
```

Reply:
Totally makes sense. Thanks! I replaced `pub` with `pub(super)` where I think it makes sense.